Module parse failed: Unexpected token

**Describe the bug**

It works fine in the 2.0.0-alpha.41, but the recent 2.0.1 introduced the following error during the build
```
 ERROR  Failed to compile with 1 error                                  20:25:31

 error  in ./node_modules/@vuelidate/core/dist/index.mjs

Module parse failed: Unexpected token (752:28)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|         resultsCache,
|         globalConfig,
>         instance: instance ?? {},
|         externalResults: $externalResults
|       });

 @ ./node_modules/cache-loader/dist/cjs.js??ref--13-0!./node_modules/thread-loader/dist/cjs.js!./node_modules/babel-loader/lib!./node_modules/cache-loader/dist/cjs.js??ref--1-0!./node_modules/vue-loader-v16/dist??ref--1-1!./src/App.vue?vue&type=script&lang=js 2:0-43 17:15-27
 @ ./src/App.vue?vue&type=script&lang=js
 @ ./src/App.vue
 @ ./src/main.js
 @ multi ./src/main.js

 ERROR  Build failed with errors.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vuelidate-2-composition-api-vue-3@0.1.0 build: `vue-cli-service build`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the vuelidate-2-composition-api-vue-3@0.1.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Reproduction URL**
used original sample with just updated version of the library
https://codesandbox.io/s/competent-phoebe-sxjyjz?file=/package.json


**To Reproduce**
Steps to reproduce the behavior:
1. use `2.0.1` version of the package
2. run `npm install`
3. run `npm run build`
4. See error

**Expected behavior**
No error


**Additional context**
Node version:
v14.20.1

--- a/babel.config.js
+++ b/babel.config.js
@@ -5,7 +5,8 @@ module.exports = {
     }]
   ],
   plugins: [
-    '@babel/plugin-proposal-object-rest-spread'
+    '@babel/plugin-proposal-object-rest-spread',
+    '@babel/plugin-proposal-nullish-coalescing-operator'
   ],
   comments: false,
   env: {

